neo.UnitTests/UT_ContractManifest.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Neo.Cryptography.ECC;
+using Neo.SmartContract.Manifest;
+
+namespace Neo.UnitTests
+{
+    [TestClass]
+    public class UT_ContractManifest
+    {
+        [TestMethod]
+        public void ParseFromJson_Default()
+        {
+            var json = @"{""groups"":[],""features"":{""storage"":false,""payable"":false},""abi"":{""hash"":""0x0000000000000000000000000000000000000000"",""entryPoint"":{""name"":""Main"",""parameters"":[{""name"":""operation"",""type"":""String""},{""name"":""args"",""type"":""Array""}],""returnType"":""Any""},""methods"":[],""events"":[]},""permissions"":[{""contract"":""*"",""methods"":""*""}],""trusts"":[],""safeMethods"":[]}";
+            var manifest = ContractManifest.Parse(json);
+
+            Assert.AreEqual(manifest.ToString(), json);
+            Assert.AreEqual(manifest.ToString(), ContractManifest.CreateDefault(UInt160.Zero).ToString());
```

Comment:
This CreateDefault was a nice simplification, @shargon. It became easy to follow the UT's. Nice job.